lib/rubocop/cop/naming/predicate_name.rb

```diff
@@ -151,6 +189,10 @@ def predicate_prefixes
           cop_config['NamePrefix']
         end
 
+        def sorbet?
```

Comment:
```suggestion
        def use_sorbet_sigs?
```

This would be clearer IMO.